### PR TITLE
Drop remarks on setting the content root

### LINKED ISSUE #13057: Possible mistake in content root path setting example
If we use code sample from example, line ``builder.UseSolutionRelativeContentRoot("<SOLUTION-RELATIVE-PATH>");`` won't be called:
```
public class CustomWebApplicationFactory<TStartup>
     : WebApplicationFactory<RazorPagesProject.Startup>
{
     protected override void ConfigureWebHost(IWebHostBuilder builder)
     {
         builder.ConfigureServices(services =>
         {
             //This code is not called
             builder.UseSolutionRelativeContentRoot("<SOLUTION-RELATIVE-PATH>");
             ...
         });
     }
}
```
But if we write this line directly in overriden method, this will override content root path correctly:

```
public class CustomWebApplicationFactory<TStartup>
     : WebApplicationFactory<RazorPagesProject.Startup>
{
     protected override void ConfigureWebHost(IWebHostBuilder builder)
     {
        //Now this works
        builder.UseSolutionRelativeContentRoot("<SOLUTION-RELATIVE-PATH>");
     }
}
```

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 77889d43-fedb-f8b4-8fea-379ec946555c
* Version Independent ID: f499b131-8b14-7e10-1728-543fdd5a5656
* Content: [Integration tests in ASP.NET Core](https://docs.microsoft.com/en-us/aspnet/core/test/integration-tests?view=aspnetcore-2.2#how-the-test-infrastructure-infers-the-app-content-root-path)
* Content Source: [aspnetcore/test/integration-tests.md](https://github.com/aspnet/AspNetCore.Docs/blob/master/aspnetcore/test/integration-tests.md)
* Product: **aspnet-core**
* Technology: **aspnetcore-test**
* GitHub Login: @guardrex
* Microsoft Alias: **riande**

### PR DESCRIPTION
**Pull request by guardrex**
Fixes #13057

### CODE REVIEW

**Comment by guardrex** [comment]
@javiercn This might be a bug fix. It will only take 10 seconds to confirm with you.

File: aspnetcore/test/integration-tests.md [hunk 1]
```diff
@@ -295,12 +295,9 @@ Call the [UseSolutionRelativeContentRoot](/dotnet/api/microsoft.aspnetcore.testh
    {
        protected override void ConfigureWebHost(IWebHostBuilder builder)
        {
-           builder.ConfigureServices(services =>
-           {
-               builder.UseSolutionRelativeContentRoot("<SOLUTION-RELATIVE-PATH>");
+           builder.UseSolutionRelativeContentRoot("<SOLUTION-RELATIVE-PATH>");
```
Comment (javiercn) [inline aspnetcore/test/integration-tests.md]: I'm confused, why are recommending this in the first place? The WAF base class should take care of this automatically.

Comment (guardrex) [reply aspnetcore/test/integration-tests.md]: We say this today ...

> In most cases, it isn't necessary to explicitly set the app content root, as the search logic usually finds the correct content root at runtime. In special scenarios where the content root isn't found using the built-in search algorithm, the app content root can be specified explicitly or by using custom logic. To set the app content root in those scenarios, call the `UseSolutionRelativeContentRoot` extension method ...

Would you like to drop all of that + the code example?

Comment (javiercn) [reply aspnetcore/test/integration-tests.md]: I think so.

Comment (guardrex) [reply aspnetcore/test/integration-tests.md]: Thanks @javiercn ... I hope to be free-er-ish later today, and I'll get this squared away.

**Comment by guardrex** [comment]
@Rick-Anderson Approval is here :point_right: https://github.com/aspnet/AspNetCore.Docs/pull/13058#discussion_r310690947

**Pull request merged**